### PR TITLE
Updating DNS config call out

### PR DESCRIPTION
**Pull request by kennyj42**
Updating to include partial CNAME setup

### CODE REVIEW

File: content/cloudflare-one/applications/configure-apps/_index.md [hunk 1]
```diff
@@ -14,6 +14,6 @@ You can protect two types of web applications: SaaS and self-hosted.
...
 
-* [**Self-hosted applications**](/cloudflare-one/applications/configure-apps/self-hosted-apps/) consist of internal applications that you host in your own environment. These can be the data center versions of tools like the Atlassian suite or applications created by your own team. To secure self-hosted applications, you must use Cloudflare's authoritative DNS and [connect the application](/cloudflare-one/connections/connect-apps/) to Cloudflare.
+* [**Self-hosted applications**](/cloudflare-one/applications/configure-apps/self-hosted-apps/) consist of internal applications that you host in your own environment. These can be the data center versions of tools like the Atlassian suite or applications created by your own team. To secure self-hosted applications, you must use Cloudflare's authoritative DNS (partial [CNAME configurations](/dns/zone-setups/partial-setup/)  may also be used) and [connect the application](/cloudflare-one/connections/connect-apps/) to Cloudflare.
```
Comment (kodster28) [inline content/cloudflare-one/applications/configure-apps/_index.md]:
```suggestion
* [**Self-hosted applications**](/cloudflare-one/applications/configure-apps/self-hosted-apps/) consist of internal applications that you host in your own environment. These can be the data center versions of tools like the Atlassian suite or applications created by your own team. To secure self-hosted applications, you must use Cloudflare's DNS ([full setup](/dns/zone-setups/full-setup/) or [partial CNAME setup](/dns/zone-setups/partial-setup/)) and [connect the application](/cloudflare-one/connections/connect-apps/) to Cloudflare.
```

**Review by kodster28 — APPROVED** [review]
I'd suggest rewording it slightly and also including the full setup link, but lgtm overall

File: content/cloudflare-one/applications/configure-apps/_index.md (hunk 1 above)
Comment (ranbel) [inline content/cloudflare-one/applications/configure-apps/_index.md:17]:
```suggestion
* [**Self-hosted applications**](/cloudflare-one/applications/configure-apps/self-hosted-apps/) consist of internal applications that you host in your own environment. These can be the data center versions of tools like the Atlassian suite or applications created by your own team. To secure self-hosted applications, you must use [Cloudflare as a DNS provider](/dns/zone-setups/) and [connect the application](/cloudflare-one/connections/connect-apps/) to Cloudflare's edge.
```

**Review by ranbel — APPROVED** [review]

**Review by kennyj42 — COMMENTED** [review]
Agreed with that!
